README.md

```diff
@@ -1,2 +1,14 @@
-# github-profiles-automator
-Github profiles automator charm
+# GitHub Profiles Automator
+
+[![CI](https://github.com/canonical/github-profiles-automator/actions/workflows/integrate.yaml/badge.svg)](https://github.com/canonical/github-profiles-automator/actions/workflows/integrate.yaml)
+[![On Pull Request](https://github.com/canonical/github-profiles-automator/actions/workflows/on_pull_request.yaml/badge.svg)](https://github.com/canonical/github-profiles-automator/actions/workflows/on_pull_request.yaml)
+[![On Push](https://github.com/canonical/github-profiles-automator/actions/workflows/on_push.yaml/badge.svg)](https://github.com/canonical/github-profiles-automator/actions/workflows/on_push.yaml)
+[![Publish](https://github.com/canonical/github-profiles-automator/actions/workflows/publish.yaml/badge.svg)](https://github.com/canonical/github-profiles-automator/actions/workflows/publish.yaml)
+[![Release charm to other tracks and channels](https://github.com/canonical/github-profiles-automator/actions/workflows/release.yaml/badge.svg)](https://github.com/canonical/github-profiles-automator/actions/workflows/release.yaml)
+
+Github Profiles Automator charm is focused on updating Profiles and
+RoleBindings / AuthorizationPolicies for contributors in a Kubeflow
+cluster based on a central source of truth, a ProjectManagementRepresentation
+(PM) defined in a file in a GitHub repo.
+
+For more information please read our [official spec](https://docs.google.com/document/d/1WlRJTOs-J1ghZVygO3AOFvbETLyt_fY2h_PsafPMLa0/edit?tab=t.0)
```

Comment:
Hmm I had the impression that by default all design docs are public, since the code is as well.

I liked the idea since the spec gives the more technical details on the implementation, so it made more sense to live closer to the repo. Let's also discuss with the rest of the team.

Am OK as well to remove and link to docs in the future

Reply:
Removing for now, let's add a link to the docs in the future
ba97efe

Reply:
Created https://github.com/canonical/github-profiles-automator/issues/15, and added daily note